packages/vitest/src/utils/index.ts

```diff
@@ -1,3 +1,4 @@
+import crypto from 'node:crypto'
```

Comment:
Ah, actually. This file can be imported in any environment. The utility should be in node-specific utils file

Reply:
Should I create a separate file or does such a file already exist and I need to move this function into it?

Reply:
Creating a separate file should be fine

Reply:
Ok.